src/components/table.js

```diff
@@ -683,9 +688,10 @@ export const ColumnSelector = ({ onSave, columnSettings }) => {
 
   return h(Fragment, [
     h(Clickable, {
-      'aria-label': 'Select columns',
```

Comment:
Oh. Right. I pulled this out because #2525 will make it redundant; the tooltip will be used as a label once that branch is merged. At one time those changes were in this branch...

I can restore that line for now, to keep the two PRs entirely independent. Or leave it like this assuming they'll be merged at roughly the same time.  Your choice. :)

Reply:
Nevermind. The change is already in that PR. I'll revert it here